DataCite.js

```diff
@@ -105,9 +105,11 @@ var testCases = [
 				"seeAlso": [],
 				"attachments": [],
 				"title": "Code criminel de l'empereur Charles V vulgairement appellé la Caroline contenant les loix qui sont suivies dans les jurisdictions criminelles de l'Empire et à l'usage des conseils de guerre des troupes suisses.",
-				"url": "http://dx.doi.org/10.12763/ONA1045",
-				"DOI": "10.12763/ONA1045",
```

Comment:
The deletion of the DOI for a report is here triggered by `ZU.itemFromCSLJSON` (I guess this [line](https://github.com/zotero/zotero/blob/master/chrome/content/zotero/xpcom/utilities.js#L1879)).

However, we can also add the DOI in `extra` again in the DataCite translator. I will do this later...